src/BenchmarkDotNet/EventProcessors/EventProcessorBase.cs

```diff
@@ -5,15 +5,15 @@
 using BenchmarkDotNet.Toolchains.Results;
 using BenchmarkDotNet.Validators;
 
-namespace BenchmarkDotNet.EventHandlers
+namespace BenchmarkDotNet.EventProcessors
 {
-    public abstract class BenchmarkEventHandlerBase
+    public abstract class EventProcessorBase
```

Comment:
in the past we used the `Base` suffix for abstract types, but from what I've learned from API Reviews it's an anti-pattern

```suggestion
    public abstract class EventProcessor
```